EIPS/eip-3712.md

```diff
@@ -0,0 +1,337 @@
+---
+eip: 3712
+title: Standard for Multiple Types of Fungible-Tokens
+description: A standard that manages multiple types of fungible tokens in a single contract.
```

Comment:
We try to discourage standard (and similar words) in the title and description fields. They tend to show up in every EIP otherwise.